modules/mobile-agents/android/client/src/org/wso2/mdm/agent/GCMIntentService.java

```diff
@@ -63,13 +63,17 @@ protected void onUnregistered(Context context, String registrationId) {
 	protected void onMessage(Context context, Intent intent) {
 		String mode =
 				Preference.getString(context,
-				                     context.getResources()
-				                            .getString(R.string.shared_pref_message_mode)
+						context.getResources()
+								.getString(R.string.shared_pref_message_mode)
```

Comment:
This file has been removed in Commit [https://github.com/wso2/product-mdm/commit/b0c66506d24fb5288300bec4224f1eaa58c7b426]